README.md

```diff
@@ -0,0 +1,64 @@
+<p align="center">
+  <img src="/images/banner.png">
+</p>
+
+**Gong** is the [Xmartlabs](https://xmartlabs.com/)' official Android template project, written in Kotlin and focused on provides a solid app's architecture.
+One of the main objectives of this project is to supply you with a good starting point for all new android apps, which let you move forward fast using the latest Android Components and libraries.
+We're using "clean architecture" to structure, decouple, expand, and maintain the code.
```

Comment:
maybe a link to a good clean architecture blog you may know :)

Reply:
maybe we can make the blog ;) haha

Reply:
I'll add it later